TAF_EasyRest/TestFramework/PageComponents/NavigationMenuPageComponents/NavigationMenuPageComponent.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace TestFramework.PageComponents.NavigationMenuComponents
+{
+    public class NavigationMenuPageComponent : BasePage
+    {
+        public NavigationMenuPageComponent(IWebDriver driver) : base(driver)
+        {
+            wait = new WebDriverWait(driver, timeout);
+        }
+
+        protected static TimeSpan timeout = TimeSpan.FromSeconds(3);
```

Comment:
This is what is supposed to be on the test level, page object doesn't know what timeout should be for searching elements.

Reply:
removed instances of WebDriverWait class from all pages